README.md

```diff
@@ -51,6 +52,10 @@ Physical path formats supported by guard tool:-
 * /sys-0/node-0/proc-0/mc-0/mi-0/mcc-0
 * sys-0/node-0/proc-0/mc-0/mi-0/mcc-0
 
+**Note:**
```

Comment:
Looks you did not change here i.e using as old
Also, you used bullet but, not showing as a bullet - please correct if planning to change.